src/pages/Flavors.test.js

```diff
@@ -139,126 +152,92 @@ describe('Page <Flavors />', () => {
       }
     }
   ];
-  const pager = { count: 5, limit: 20, page: 1, pages: 1 };
-  const store = mockStore({
-    actions,
-    application: appInitialState,
-    flavor: flavorInitialState,
-    flavors: flavorsInitialState
-  });
 
+  const pager = { count: 5, limit: 20, page: 1, pages: 1 };
   const pagerNavigation = <p></p>;
 
-  const notLoggedInProps = {
-    actions,
+  const props = {
     collection,
-    loggedIn: false,
     pager,
-    pagerNavigation,
-    stash,
-    stashLoaded: false
+    pagerNavigation
   };
 
-  const noStashProps = {
-    actions,
-    collection,
-    loggedIn: true,
-    pager,
-    pagerNavigation,
-    stash,
-    stashLoaded: false
-  };
+  const ReduxConnectedFlavors = (flavorsProps) => (
+    <Provider store={store}>
+      <Flavors {...flavorsProps} />
+    </Provider>
+  );
+
+  beforeEach(() => {
+    useDispatch().mockClear();
+    isLoggedIn.mockClear();
+    getStash.mockClear();
+    isLoaded.mockClear();
+  });
 
-  const props = {
-    actions,
-    collection,
-    loggedIn: true,
-    pager,
-    pagerNavigation,
-    stash,
-    stashLoaded: true
-  };
+  it('has "log in" call to action when user is not logged in', () => {
+    isLoggedIn.mockReturnValue(false);
+    getStash.mockReturnValue([]);
+    isLoaded.mockReturnValue(false);
 
-  const RoutedConnectedFlavors = withMemoryRouter(ConnectedFlavors);
-  const RoutedFlavors = withMemoryRouter(Flavors);
+    const { getByText } = render(<ReduxConnectedFlavors {...props} />);
 
-  it('renders correctly', () => {
-    expect(
-      renderer
-        .create(
-          <Provider store={store}>
-            <RoutedConnectedFlavors />
-          </Provider>
-        )
-        .toJSON()
-    ).toMatchSnapshot();
+    expect(getByText(/log in/i)).toBeDefined();
```

Comment:
I think this should be `.toBeInTheDocument()`, no?

Reply:
I think you're right. I'll mess with it.

Reply:
yeah, you're right. I was trying that earlier, but I think I was using `toBeInDocument` w/o `The`, so it wasn't working right.. thanks for catching this